### PR TITLE
Release 3.2.0 updates

### PR DESCRIPTION
**Pull request by thejcfactor**

### CODE REVIEW

**Review by RichardSmedley — COMMENTED** [review]
Thanks for all of this @thejcfactor - LGTM :+1:  - 
leaving unmerged this morning to give a chance for another pair of eyes on this.

File: modules/hello-world/pages/sample-application.adoc [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 :content-type: tutorial
 :page-topic-type: tutorial
 :page-aliases: ROOT:sample-application,ROOT:tutorial4,ROOT:sample-app-backend
 
 [abstract]
-{description}
+{description}
 
-include::6.6@sdk:shared:partial$sample-application.adoc[tag=prereq]
+include::7.06@sdk:shared:partial$sample-application.adoc[tag=prereq]
```
Comment (maria-robobug) [inline modules/hello-world/pages/sample-application.adoc]: Think this should be `7.0`

File: modules/howtos/pages/encrypting-using-sdk.adoc [hunk 2]
```diff
@@ -1,15 +1,175 @@
 = Field Level Encryption from the SDK
-:description: The Field Level Encryption library enables encryption and decryption of JSON fields, to support FIPS-140-2 compliance.
+:description: A practical guide for getting started with Field-Level Encryption, showing how to encrypt and decrypt JSON fields using the Python SDK.
 :page-topic-type: howto
 :page-edition: Enterprise Edition
-:page-status: Developer Preview
-// // :page-aliases: ROOT:encrypting-using-sdk.adoc
+//:page-aliases: ROOT:encrypting-using-sdk.adoc
```
Comment (maria-robobug) [inline modules/howtos/pages/encrypting-using-sdk.adoc]: Think we need to un-comment this @RichardSmedley?

File: modules/howtos/pages/encrypting-using-sdk.adoc [hunk 3]
```diff
@@ -1,15 +1,175 @@
...
-Client-side implementation of Field Level Encryption is available in the previous version of the SDK.
-It will be enabled in a future release of the third generation SDK.
+[#package]
+== Packaging
 
-A _Developer Preview_ is available in the xref:3.0@java-sdk:howtos:encrypting-using-sdk.adoc[Java SDK].
+The Couchbase Python SDK works together with the https://github.com/couchbase/python-couchbase-encryption[Python Couchbase Encryption^] library to provide support for encryption and decryption of JSON fields.
+This library includes cryptographic algorithms you can use out of the box, and provides a framework for implementing your own crypto components.
+
+NOTE: The encryption code is packaged as an optional library because it is released under a https://github.com/couchbase/python-couchbase-encryption/blob/master/LICENSE[different license^] than the Couchbase Python SDK.
```
Comment (maria-robobug) [inline modules/howtos/pages/encrypting-using-sdk.adoc]: We had a discussion with Matt about this, the note might need to change a little bit as we need to refer to the `Licence` and `Enterprise Subscription License` it seems:

See https://github.com/couchbase/docs-sdk-go/pull/136/files as an example (though still pending review), @RichardSmedley is it ok for this to be updated as I've done in the linked PR?

**Review by maria-robobug — APPROVED** [review]
👍 Awesome stuff! Just had some minor questions/comments.
I noticed we're using default collections for these examples, I assume we're updating them to use named scopes and collections in a different PR/ticket?

File: modules/howtos/pages/encrypting-using-sdk.adoc (hunk 3 above)
Comment (RichardSmedley) [reply modules/howtos/pages/encrypting-using-sdk.adoc]: Thanks @maria-robobug - yes, please update.  :+1:

File: modules/howtos/pages/encrypting-using-sdk.adoc (hunk 2 above)
Comment (RichardSmedley) [reply modules/howtos/pages/encrypting-using-sdk.adoc]: done

File: modules/hello-world/pages/sample-application.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/hello-world/pages/sample-application.adoc]: fixed :+1:

File: modules/howtos/pages/encrypting-using-sdk.adoc (hunk 3 above)
Comment (maria-robobug) [reply modules/howtos/pages/encrypting-using-sdk.adoc]: Done.

**Comment by thejcfactor** [comment]
@maria-robobug  - good point!  For FLE, would it be fair to revisit the docs once transcoders/serializers are implemented (hoping by 3.2.1, but it might be 3.2.2)?  I can make sure they move to referencing scopes/collections at that point.  It is an easy enough update if we want to change it now though.  Just lemme know. 👍

**Pull request merged**